internal/annotation/annotations.go

```diff
@@ -24,6 +24,7 @@ import (
 )
 
 // DEFAULT_INGRESS_CLASS_NAME is the Contour default.
+// nolint:revive
```

Comment:
FYI this was replaced by a differently-named const that doesn't need a `// nolint` as part of #3902

Reply:
I removed this line @skriss